src/uct/ib/dc/base/dc_iface.c

```diff
@@ -307,3 +307,180 @@ ucs_status_t uct_dc_iface_flush(uct_iface_h tl_iface, unsigned flags, uct_comple
     return status;
 }
 
+ucs_status_t uct_dc_iface_init_fc_ep(uct_dc_iface_t *iface, int ep_size)
+{
+    ucs_status_t status;
+    uct_dc_ep_t *ep;
+
+    ep = ucs_malloc(ep_size, "fake_fc_ep");
+    if (ep == NULL) {
+        ucs_error("Failed to allocate fake FC ep");
+        status =  UCS_ERR_NO_MEMORY;
+        goto err;
+    }
+    /* We do not have any peer address at this point, so init basic subclasses
+     * only (for statistics, iface, etc) */
+    status = UCS_CLASS_INIT(uct_base_ep_t, (void*)(&ep->super),
```

Comment:
For mlx5 it would be fine I guess.
But for verbs I wanted to reuse existing API for posting send.  Looks like there is no other need for that, so I can try to use DC EP and add some wrapper to VERBS posting API